content/en/design/core/_index.md

```diff
@@ -73,152 +55,64 @@ These are the status indication colors of the system. When necessary, use white
-| **Use bold to add extra emphasis to items of importance** <br>This includes patient/contact names and target titles. 	| **Do not overuse italics**<br>Aside from Enketo forms where it is used to differentiate explanatory text or suggested actions, do not use italics. 	|
-| **Use underlines for links** <br>The general usability association is that underlined text is a hyperlink. 	| **Avoid using both bold and italic at the same time** <br>If you want to add emphasis to something, consider large size text in bold.	|
-
-
-
-### Base typeface
-
-```
-font-family: Noto, sans-serif
-font-size: 16px (1rem)
-line-height: 1.375rem (22px)
-color: #333333
-```
-
-### Type sizes
-
 H1 is the highest hierarchical level of text, and should be used sparingly. It is used for the large text underneath percentage bars.
 
+H2 is used as a header style for main content sections on the right hand side, such as a task title, the name of a person/place on their profile, or the title of a targets widget.
 
-```
-H1
-font-size: 24px (1.5rem)
-line-height: 2rem (32px)
-```
+H3 is used for titles of condition cards and section titles on the form summary page.
 
-<br> 
-
-![H1](img/H1.png)
-
-<br> 
-
-<br>
-
-H2 is used as a header style for main content sections on the right hand side, such as a task title, the name of a person/place on their profile, or the title of a targets widget. 
-
-```
-H2 
-font-size: 20px (1.25rem)
-Line-height: 1.75rem (26 px)
-```
-
-<br> 
-
-![H2](img/H2.png)
-
-![H2](img/H2.2.png)
-
-![H2](img/H2.3.png)
-
-
-<br> 
-
-<br>
-
-H3 is the next lowest level, used for things like the titles of condition cards and section titles on the form summary page. 
-
-```
-H3 
-font-size: 18px (1.125rem)
-line-height: 1.5rem (24px)
-```
-
-<br> 
-
-![H3](img/H3.png)
-
-![H3](img/H3.2.png)
-
-<br> 
-
-<br>
-
-H4 is the default type size, and should be used for all normal body text throughout the app. Most text should be H4 in size - **when in doubt, use H4.**
-
-```
-H4 (body)
-font-size: 16px (1rem)
-line-height: 1.375rem (22px)
-```
-<br> 
-
-![H4](img/H4.png)
-
-![H4](img/H4.2.png)
-
-<br> 
-
-<br>
+H4 is the default type size, and should be used for all normal body text throughout the app. Most text should be H4 in size. When in doubt, use H4.
 
 H5 is a smaller body text size that we use sparingly in places where space is tight, such as timestamps in the upper right of content rows, condition card filter text, “belongs to” breadcrumbs, and targets goal labels.
 
-```
-H5 (small body)
-font-size: 14px (.875rem)
-line-height: 1.25rem (20px)
-```
-<br> 
-
-![H5](img/H5.png)
+        {{% pen id="PoZObmY" %}}
```

Comment:
If the shortcode to use the id as the first param, then you could update all the shortcodes accordingly:
```suggestion
        {{% pen PoZObmY %}}
```